src/commonMain/kotlin/fr/acinq/lightning/wire/OfferTypes.kt

```diff
@@ -782,7 +782,7 @@ object OfferTypes {
              * Create an offer using a single-hop blinded path going through our trampoline node.
              *
              * @param amount amount if it can be determined at offer creation time.
-             * @param description description of the offer (may be null if [amount] is also null).
+             * @param description description of the offer (maybe null if [amount] is also null).
```

Comment:
This is not a typo though? "(it) may be null if [amount] is also null"

Reply:
ops! you are right :)